Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -148,9 +149,10 @@ @implementation WKContentView {
     RetainPtr<NSUndoManager> _undoManager;
     RetainPtr<WKQuirkyNSUndoManager> _quirkyUndoManager;
 
-    std::unique_ptr<BinarySemaphore> _pdfPrintCompletionSemaphore;
+    Lock _pendingBackgroundPrintFormattersLock;
+    RetainPtr<NSMutableSet> _pendingBackgroundPrintFormatters;
```

Comment:
could/should we `WTF_GUARDED_BY_LOCK(_pendingBackgroundPrintFormattersLock)`?

Reply:
I doubt this works in ObjC.

Reply:
It doesn't, I tried it in an initial version of this patch.